forum/qa-plugin/report-reason/q2apro-flag-reasons-layer.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+class qa_html_theme_layer extends qa_html_theme_base
+{
+
+    public function head_script()
+    {
+        qa_html_theme_base::head_script();
+        $logged = qa_is_logged_in();
```

Comment:
skoro już w każdej metodzie jest $logged = qa_is_logged_in() to może warto zrobić zmienną prywatną:
private $isLogged, którą w konstruktorze tej klasy można ustawić na wynik działania tej metody? Po co odpalać pierdylion razy tę samą metodę